pyproject.toml

```diff
@@ -11,7 +11,7 @@ packages = [{include = "django_api_decorator"}]
 
 [tool.poetry.dependencies]
 python = "^3.11"
-Django = ">=3"
+Django = "^4.2.16"
```

Comment:
I'm all for not support v. 3, and bumping it. But I think it makes sense to have a pretty broad acceptance for a library like this. I.e., `^4.2.16` doesn't allow Django 5, which I kinda think it should.

Reply:
Ahh, right, that's how it works 😬 Forgot about that 🤔